### PR TITLE
docs(button): update docs

### PR DESCRIPTION
**Pull request by nikkimk**
## What I did

1. Updated button docs #925 .

## Testing Instructions

1. View [DP button docs](https://deploy-preview-1006--red-hat-design-system.netlify.app/elements/button/).
2. Compare to [Button doc](https://docs.google.com/document/d/1t_bj0VzaMrSKcPkZGeN-aKUM9NdPjpiG1jjdzZjwWIM/edit?usp=sharing) and to [Docs cleanup checklist](https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist).

## Notes to Reviewers
The following are WIP dependent on other issues/PRs:
- [Image width styling](https://github.com/RedHat-UX/red-hat-design-system/pull/891)
- [Code tab styling](https://github.com/RedHat-UX/red-hat-design-system/issues/969)

### CODE REVIEW

**Comment by bennypowers** [comment]
see #1005 so we can reduce the size of the repo and maintenance burden

**Comment by nikkimk** [comment]
> #1005

@bennypowers Should I begin doing this now in this PR or do we want to do a batch find an replace in a branch from #1005?

File: elements/rh-button/rh-button.ts [hunk 1]
```diff
@@ -10,10 +10,12 @@ import { BaseButton } from '@patternfly/elements/pf-button/BaseButton.js';
 import styles from './rh-button.css';
 
 /**
- * @summary Performs an action in the background when triggered
+ * @summary Triggers actions on the page or in the background
+ * @description A button communicates and triggers available actions when clicked or selected. Buttons allow users to perform an action when triggered. They feature a text label, a background or a border, and icons.
```
Comment (nikkimk) [inline elements/rh-button/rh-button.ts:14]: @bennypowers any idea why this isn't showing up in the Overview tab?

File: package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "Kelsey Swanson <kswanson@redhat.com>",
     "Marionne Patel <mapatel@redhat.com>",
     "Michael Potter <mpotter@redhat.com",
+    "Nikki Massaro Kauffman <nmassaro@redhat.com>",
```
Comment (bennypowers) [inline package.json:21]: :partying_face:

**Comment by coreyvickery** [comment]
@nikkimk Tiny edits:

### Overview page
- Under the `Sample component` h2, let's go with one Primary button only
- Repo status table does not reflect the design

### Style page
- Under the `Space` h2, the word `Guidelines` should link to the Guidelines tab

**Review by coreyvickery — APPROVED** [review]
Good to go.

**Pull request merged**